console/client/src/features/timeline/TimelineCall.tsx

```diff
@@ -2,46 +2,50 @@ import { ArrowRightOnRectangleIcon } from '@heroicons/react/24/outline'
 import { Call } from '../../protos/xyz/block/ftl/v1/console/console_pb'
 import { formatDuration, formatTimestamp } from '../../utils/date.utils'
 import { panelColor, textColor } from '../../utils/style.utils'
-
+import { classNames } from '../../utils/react.utils'
 type Props = {
   call: Call
+  selected?: boolean
 }
 
-export const TimelineCall: React.FC<Props> = ({ call }) => {
+export const TimelineCall: React.FC<Props> = ({ call, selected }) => {
   return (
     <>
       <div className={`relative flex h-6 w-6 flex-none items-center justify-center ${panelColor}`}>
         <ArrowRightOnRectangleIcon className='h-6 w-6 text-indigo-500'
           aria-hidden='true'
         />
       </div>
-      <div className={`flex-auto py-0.5 text-xs leading-5 ${textColor}`}>
-        {call.destinationVerbRef && (
-          <div className={`inline-block rounded-md dark:bg-gray-700/40 px-2 py-1 mr-1 text-xs font-medium text-gray-500 dark:text-gray-400 ring-1 ring-inset ring-black/10 dark:ring-white/10`}>
-            {call.destinationVerbRef?.module}:{call.destinationVerbRef?.name}
-          </div>
-        )}
+      <div className={classNames(`relative flex gap-x-4 flex-auto w-full max-w-full p-1.5 `, selected && 'bg-indigo-600 rounded-md')}>
```

Comment:
We might want a lighter background for `light` mode here. Also the text might need to be adjusted when the row is highlighted to avoid the dark text on dark background here:
<img width="541" alt="Screenshot 2023-08-24 at 1 41 53 PM" src="https://github.com/TBD54566975/ftl/assets/51647/47ceacdb-a5f2-4c4e-8136-87e1cde0e585">

Reply:
Looks like I missed some text styles will fix

Reply:
fixed